thirdparty/fluidsynth/fluidsynth-2.1.4/src/synth/fluid_voice.c

```diff
@@ -196,7 +196,11 @@ static void fluid_voice_initialize_rvoice(fluid_voice_t *voice, fluid_real_t out
     fluid_voice_update_modenv(voice, FALSE, FLUID_VOICE_ENVFINISHED,
                               0xffffffff, 0.0f, 0.0f, -1.0f, 1.0f);
 
+#ifdef ENABLE_DEFAULT_COMPRESSION
```

Comment:
That's not really `Disable built-in Fluid IIR filter via environmental variable`, but rather via pre-processor...
Can't we have an API (or really an environment variable) for this?

Why has it been disabled in the first place? I see no reason being mentioned anywhere.

Please check https://github.com/Jojo-Schmitz/MuseScore/issues/438#issuecomment-2617832287